### DISCUSSION TITLE
use chatgpt for translations

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#211] [Show and tell]
you know the json file we use has some strange syntax ([ICU](https://unicode-org.github.io/icu/userguide/format_parse/messages/)), and people trying to produce translations are often failing at tackling it.

a user needed a Malay translation for HFS.
I don't know malay, but i used chatgpt that proved to be a really good tool that had no problem at producing correct syntax. I can't judge for the quality of the translation, tho, but when I tried the same for italian it did a good job.
This is the result I'm talking about, and I plan to include it in next release.
[hfs-lang-ms.json.zip](https://github.com/rejetto/hfs/files/11214183/hfs-lang-ms.json.zip)

how cool is that?

**Comment by GeorgeGeorgeGe** [comment]
Perfect!
